timingwheel.go

```diff
@@ -160,8 +171,38 @@ func (tw *TimingWheel) Start() {
 // not wait for the task to complete before returning. If the caller needs to
 // know whether the task is completed, it must coordinate with the task explicitly.
 func (tw *TimingWheel) Stop() {
+	if tw.IsStopped() {
+		return
+	}
 	close(tw.exitC)
 	tw.waitGroup.Wait()
+	tw.clear()
+}
+
+func (tw *TimingWheel) clear() {
+	tw.queue.Clear()
+	tw.mu.Lock()
+	tw.buckets = nil
```

Comment:
We also need to flush the Bucket as well. Otherwise, the bucket is still there.

Reply:
Thanks reminding @vietnakid 👍🏻